core/primitives/src/shard_layout.rs

```diff
@@ -536,6 +539,7 @@ impl ShardLayout {
     /// TODO(resharding) Determine the shard layout for v4.
     /// This layout is provisional, the actual shard layout should be determined
     /// based on the fresh data before the resharding.
+    #[cfg(feature = "test_utils")]
```

Comment:
Technically allow unused would be more appropriate but alright ;)

Reply:
Good catch, it is only used in tests in the same file, change to `cfg(test)`